src/components/Editor/CustomModules/FileInput/FileInput.tsx

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable */
+import React from 'react';
+import Dropzone from 'react-dropzone';
+import Paper from '@material-ui/core/Paper';
+import CropOriginalIcon from '@material-ui/icons/CropOriginal';
+import { Typography } from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+import { useStyles } from './FileInput.style';
+import { IFileInputProps } from '../types';
+import { langTokens } from '../../../../locales/localizationInit';
+
+export const FileInput: React.FC<IFileInputProps> = ({ onChange, name }) => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+
+  return (
+    <Dropzone>
+      {({ getRootProps, getInputProps }) => (
```

Comment:
Why not use hooks for it? Render prop pattern made sense before react hooks.

But hooks are more readable and a more superior pattern.